db/src/openai_utils.rs

```diff
@@ -7,15 +7,9 @@ pub async fn generate_summary(content: &str) -> Result<String> {
     openai::set_key(api_key);
 
     let messages = vec![
-        ChatCompletionMessage {
-            role: ChatCompletionMessageRole::System,
-            content: Some("You are a helpful assistant.".to_string()),
-            name: None,
-            function_call: None,
-        },
         ChatCompletionMessage {
             role: ChatCompletionMessageRole::User,
-            content: Some(format!("Summarize the following article and make sure to highlight the important parts: {content}")),
+            content: Some(format!("Provide a concise summary of the following article that can be read and understood in 2 minutes, focusing on the main points and essential details: {content}")),
             name: None,
```

Comment:
Let me know what you think about this prompt. I like better because it's a lot more short and concise. 

The User case I imagine is that someone has 5 minutes to read an article and talk to their friend about it, they don't need large paragraphs summary. They just need key highlighted bullet points.

this is an example output for this article: https://daily.dev/blog/multi-tenant-database-design-patterns-2024
<img width="915" alt="Screenshot 2024-06-12 at 12 22 37 PM" src="https://github.com/coreyja/knowledge/assets/55955558/37873f5a-0e99-4f1a-9cbb-84f2b5c3dc44">

Reply:
Love this prompt!


And definitely agree on the use case you outlined! 💯